test/vaults/BaseTest.sol

```diff
@@ -115,7 +115,7 @@ contract BaseTest is VaultsDeployer, Test {
         // Label contracts
         vm.label(address(root), "Root");
         vm.label(address(asyncRequests), "AsyncRequests");
-        vm.label(address(asyncRequests), "SyncRequests");
+        vm.label(address(syncRequests), "SyncRequests");
```

Comment:
Thanks for fixing that! Just remember I had it as part of the pricing PR but got lost during the continuous rebasing.